crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -39,21 +54,55 @@ pub struct StandardMaterial {
     /// If used together with a roughness/metallic texture, this is factored into the final base
     /// color as `metallic * metallic_texture_value`
     pub metallic: f32,
+
     #[texture(5)]
     #[sampler(6)]
     pub metallic_roughness_texture: Option<Handle<Image>>,
     /// Specular intensity for non-metals on a linear scale of [0.0, 1.0]
     /// defaults to 0.5 which is mapped to 4% reflectance in the shader
     pub reflectance: f32,
+
+    /// Used to fake the lighting of bumps and dents on a material.
+    ///
+    /// A typical usage would be faking cobblestones on a flat plane mesh in 3D.
+    ///
+    /// # Notes
+    ///
+    ///
+    /// Normal mapping with `StandardMaterial` and the core bevy PBR shaders requires:
+    /// - A normal map texture
+    /// - Vertex UVs
+    /// - Vertex tangents
+    /// - Vertex normals
+    ///
+    /// Tangents do not have to be stored in your model,
+    /// they can be generated using the [`Mesh::generate_tangents`] method.
+    /// If your material has a normal map, but still renders as a flat surface,
+    /// make sure your meshes have their tangents set.
+    ///
+    /// [`Mesh::generate_tangents`]: bevy_render::mesh::Mesh::generate_tangents
     #[texture(9)]
     #[sampler(10)]
     pub normal_map_texture: Option<Handle<Image>>,
+
     /// Normal map textures authored for DirectX have their y-component flipped. Set this to flip
     /// it to right-handed conventions.
     pub flip_normal_map_y: bool,
+
+    /// Specifies the level of exposition to indirect light of the surface of the material.
+    ///
+    /// This is usually generated and baked automatically by 3D-modelling software.
+    ///
+    /// Typically, steep concave parts of a model (such as the armpit of a shirt) are darker,
```

Comment:
```suggestion
    /// Typically, steeply concave parts of a model (such as the armpit of a shirt) are darker,
```

Reply:
imo adjective + adjective works (tall blue building), what is steep is not the concave-ness but the part, so I don't think this should be an adverb.

Reply:
Yeah I'm fine to leave it.